### PR TITLE
[C#] Add Simplify.Web #2973

### LINKED ISSUE #2973: [C#] Add Simplify.Web
Hi @i4004,

Would you mind to add (or consent that we add) [Simplify.Web](https://github.com/SimplifyNet/Simplify.Web) in `here` ?

This project compare performance based on 3 routes :

+ `GET` `/`, **SHOULD** return a successful status code and an empty string
+ `GET` `/user/:id`, **SHOULD** return a successful status code and the id
+ `POST` `/user`, **SHOULD** return a successful status code and an empty string

Regards,

### PR DESCRIPTION
**Pull request by panesofglass**
Closes #2973 

NOTE: I'm not familiar with this library, so I'm not sure that my route processing to retrieve the userId is standard or not.

### CODE REVIEW

**Comment by i4004** [comment]
@panesofglass, @waghanza  the standard way is:
```csharp
public override ControllerResponse Invoke() => Content(RouteParameters.id);
```
or
```csharp
[Get("/user/{id:int}")]
public class UserIdController : Controller
{
	public override ControllerResponse Invoke() => Content(((int)RouteParameters.id).ToString());
}
```

File: csharp/simplify.web/Controllers/UserIdController.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿using Simplify.Web;
+using Simplify.Web.Attributes;
+
+namespace web.Controllers
+{
+	[Get("/user/{id}")]
+	public class UserIdController : Controller
+	{
+		public override ControllerResponse Invoke() => Content(RouteParameters.id);
```
Comment (panesofglass) [inline csharp/simplify.web/Controllers/UserIdController.cs:9]: @i4004 is this better?

Comment (i4004) [reply csharp/simplify.web/Controllers/UserIdController.cs:9]: @panesofglass yes, it is ok.

**Pull request merged**